docs/guides/latest/volumesnapshot/deployment-backup.md

```diff
@@ -0,0 +1,262 @@
+## Backup Deployment's Volumes
+
+This tutorial will show you how to use Stash Operator to backup Deployment's volumes using [Volume Snapshot](https://kubernetes.io/docs/concepts/storage/volume-snapshots/) API resources.
+
+##### Before You Begin
+
+* At first, you need to familiarize with the GCE CSI-driver, to enable the Volume Snapshotting feature via feature gate and to install Stah Operator in your cluster. If you are not familiar with that please visit [here](docs/guides/latest/volumesnapshot/backup.md) 
+
+To keep everything isolated, we are going to use a separate namespace called `demo` throughout this tutorial.
+
+```console
+$ kubectl create ns demo
+namespace/demo created
+```
+
+Now, we will create a StorageClass for further uses. 
```

Comment:
If you don't already have a `StorageClass` that uses the CSI driver that supports `VolumeSnapshot` feature, create one first. Here, we are going to create `StorageClass` that uses GCE CSI driver.

Reply:
`Now, we will create` -> `Now, create`